projects/schematics/package.json

```diff
@@ -11,8 +11,8 @@
   "scripts": {
     "prepare:schematics": "yarn install",
     "clean": "yarn prepare:schematics && rimraf \"src/**/*.js\" \"src/**/*.js.map\" \"src/**/*.d.ts\"",
-    "build": "yarn clean && tsc -p tsconfig.json",
-    "test": "yarn build && jasmine \"src/**/*_spec.js\""
+    "build": "yarn clean && tsc -p ./tsconfig.schematics.json",
+    "test": "yarn clean && tsc -p ./tsconfig.schematics.spec.json && jest --config ./jest.schematics.config.js"
```

Comment:
Why do we need to compile schematics before running tests? I would only run here `jest` command with it's config.